testsuite/tests/win-unicode/mltest.ml

```diff
@@ -1,3 +1,33 @@
+(* TEST
+include unix
+flags += "-strict-sequence -safe-string -w A -warn-error A"
+files = "print.ml mkfiles.c"
+* windows-unicode
+** setup-ocamlc.byte-build-env
+program = "${test_build_directory}/mltest.byte"
+*** ocamlc.byte
+program = "${test_build_directory}/print.exe"
```

Comment:
@shindere This auxiliary program needs to be compiled with the `unix` library, how do I do that?